projects/inverse/index.js

```diff
@@ -0,0 +1,205 @@
+/*==================================================
+  Modules
+  ==================================================*/
+
+const _ = require("underscore");
```

Comment:
I don't think `underscore` 's performance gains are usable if the data is gathered via a Node process, but I'm not sure how DeFi Pulse uses this module exactly.

In general, I'd say to prefer the native calls over `underscore` ones for clarity. I don't mind the `underscore` implementation, though. Can stay.

Reply:
The rest of the projects all use underscore, so I tried to just stick with the same patterns to just keep it simple for reviewers.